boefjes/boefjes/plugins/kat_webpage_capture/main.py

```diff
@@ -5,14 +5,27 @@
 from typing import ByteString, Generator, List, Tuple, Union
 
 import docker
-from PIL import Image
+from PIL import Image, UnidentifiedImageError
 
 from boefjes.job_models import BoefjeMeta
 
-PLAYWRIGHT_IMAGE = "mcr.microsoft.com/playwright:v1.33.0-jammy"
+PLAYWRIGHT_IMAGE = "mcr.microsoft.com/playwright:latest"
```

Comment:
Why not just pin the latest version? We had a few discussions and bugs before when using the `latest` tag and therefore decided not to use that tag (as an unwritten rule) and always use a fixed version or hash. Also when using `latest` it doesn't guarantee that this plugin in production will use the latest image, whilst pinning the version will guarantee that a new image is downloaded if the version changes.

```suggestion
PLAYWRIGHT_IMAGE = "mcr.microsoft.com/playwright:v1.34.0-jammy"
```

Reply:
Because the error we got with the old version was that the container needed to be updated. I'm not sure how the containers are built, but if you run an old container version, an error message will appear saying that there is a newer version available that should be used instead. So there's no use in pinning a specific version, because it will break once a new version has been released.

Reply:
I've included logs and more background information here: https://github.com/minvws/nl-kat-coordination/issues/1190#issuecomment-1645573638

Reply:
So this container has two options? Either run 'latests', or don't run at all?
Seeing that we'd need to be able to show which version of each container was used for a given job, and be able to reproduce that, maybe this specific distribution is not a workable solution for us.
It also opens up the doors to supply line attacks, which is another reason to version pin dependencies.

Reply:
We can use the hash of the container to log which version was ran.

I'd say the bigger risk is that the container installs the Playwright NPM module when it starts, the NPM registry is presumably a bigger supply chain risk than Microsoft's container registry. The fact that this NPM module is into preinstalled is also what causes older container images to break in the first place.

Reply:
This one got probably lost in the comments, but why not make the version a setting of the boefje? (https://github.com/minvws/nl-kat-coordination/pull/1430#issuecomment-1647543346)

Reply:
Prevention of running a version of the container that was not tested with the release of OpenKAT

Reply:
With this container image there seem to be two options here as far as I can see:
- Use latest version that might break when a new version is released
- Use a pinned version, either in code or in settings, that is guaranteed to break when a new version is released.

I don't really understand why making sure it is always broken should be preferred above having the risk it might get broken.

Another solution might be to use a different container image / build our own.